README.md

```diff
@@ -60,12 +61,11 @@ melt restore ./recovered_id_ed25519 < words
 
 It all comes down to the private key __seed__:
 
-> Ed25519 keys start life as a 32-byte (256-bit) uniformly random binary seed (e.g. the output of SHA256 on some random input). The seed is then hashed using SHA512, which gets you 64 bytes (512 bits), which is then split into a “left half” (the first 32 bytes) and a “right half”. The left half is massaged into a curve25519 private scalar “a” by setting and clearing a few high/low-order bits. The pubkey is generated by multiplying this secret scalar by “B” (the generator), which yields a 32-byte/256-bit group element “A”.
-> https://blog.mozilla.org/warner/2011/11/29/ed25519-keys/
```

Comment:
This link is missing after the edit

Reply:
Ah, so what I did there was use a markdown footnote. But if you think it's better the current way I can totally revert it back. Here's what it looks like:

* * *

<img width="247" alt="Screen Shot 2022-03-17 at 9 11 44 PM" src="https://user-images.githubusercontent.com/25087/158918671-394d23df-fcd3-4f76-990e-d6ebceaa889c.png">

* * *

<img width="546" alt="Screen Shot 2022-03-17 at 9 13 42 PM" src="https://user-images.githubusercontent.com/25087/158918784-c67eca57-082c-41da-9cda-3defb95a801d.png">

Reply:
aah I was reviewing through the phone and missed that, looks good then, dont mind my blindness  :D